src/NuGet.Core/NuGet.Protocol/Model/IPackageSearchMetadata.cs

```diff
@@ -23,6 +23,7 @@ public interface IPackageSearchMetadata
         PackageIdentity Identity { get; }
         Uri LicenseUrl { get; }
         Uri ProjectUrl { get; }
+        Uri ReadmeUrl { get; }
```

Comment:
If the plan is too soon/eventually have it displayed from local feed, an approach such as LicenseMetadata might be preferred. 

Given @advay26's limited time left, what'd be the priority of this change?
@chgill-MSFT @zkat